pulpcore.te

```diff
@@ -129,11 +138,18 @@ libs_exec_ldconfig(pulpcore_server_t)
 
 # Needed for systemd Type=notify support
 init_write_pid_socket(pulpcore_server_t)
+allow pulpcore_server_t init_var_run_t:unix_dgram_socket { connect create };
+allow pulpcore_server_t self:unix_dgram_socket { connect create };
+allow kernel_t init_var_run_t:unix_dgram_socket sendto;
```

Comment:
This one should not be needed.

Reply:
The 3rd line?

Because the 2nd line is needed for a different test system:
```
type=PROCTITLE msg=audit(08/24/2021 16:33:40.846:766) : proctitle=/usr/local/lib/pulp/bin/python3.8 /usr/local/lib/pulp/bin/gunicorn pulpcore.app.wsgi:application --bind 127.0.0.1:24817 --worker 
type=SYSCALL msg=audit(08/24/2021 16:33:40.846:766) : arch=x86_64 syscall=socket success=yes exit=6 a0=local a1=SOCK_DGRAM a2=ip a3=0xd items=0 ppid=1 pid=3745 auid=unset uid=pulp gid=pulp euid=pulp suid=pulp fsuid=pulp egid=pulp sgid=pulp fsgid=pulp tty=(none) ses=unset comm=gunicorn exe=/opt/rh/rh-python38/root/usr/bin/python3.8 subj=system_u:system_r:pulpcore_server_t:s0 key=(null) 
type=AVC msg=audit(08/24/2021 16:33:40.846:766) : avc:  denied  { create } for  pid=3745 comm=gunicorn scontext=system_u:system_r:pulpcore_server_t:s0 tcontext=system_u:system_r:pulpcore_server_t:s0 tclass=unix_dgram_socket permissive=1 
----
type=PROCTITLE msg=audit(08/24/2021 16:33:40.846:767) : proctitle=/usr/local/lib/pulp/bin/python3.8 /usr/local/lib/pulp/bin/gunicorn pulpcore.app.wsgi:application --bind 127.0.0.1:24817 --worker 
type=SYSCALL msg=audit(08/24/2021 16:33:40.846:767) : arch=x86_64 syscall=connect success=yes exit=0 a0=0x6 a1=0x7ffe9e995450 a2=0x15 a3=0x22 items=0 ppid=1 pid=3745 auid=unset uid=pulp gid=pulp euid=pulp suid=pulp fsuid=pulp egid=pulp sgid=pulp fsgid=pulp tty=(none) ses=unset comm=gunicorn exe=/opt/rh/rh-python38/root/usr/bin/python3.8 subj=system_u:system_r:pulpcore_server_t:s0 key=(null) 
type=AVC msg=audit(08/24/2021 16:33:40.846:767) : avc:  denied  { connect } for  pid=3745 comm=gunicorn scontext=system_u:system_r:pulpcore_server_t:s0 tcontext=system_u:system_r:pulpcore_server_t:s0 tclass=unix_dgram_socket permissive=1 
----
```

Reply:
Well it's not needed, I tested that. I'll remove it.